.circleci/config.yml

```diff
@@ -34,14 +34,47 @@ jobs:
             git tag $TAG_NAME
             git push origin $TAG_NAME
 
+  integration-test:
+    docker:
+      - image: circleci/node:8.11
+
+    environment:
+      IMAGE_NAME: quay.io/broadinstitute/martha
+      QUAY_USER: broadinstitute+martha_circle_bot
+
+    steps:
+      - checkout
+
+      - setup_remote_docker:
+          docker_layer_caching: true
+
+      - run:
+          name: Build image
+          command: |
+            docker build -t ${IMAGE_NAME}:${CIRCLE_BRANCH} -f docker/Dockerfile .
+            if [[ ${CIRCLE_BRANCH} == "dev" ]]; then docker tag ${IMAGE_NAME}:${CIRCLE_BRANCH} ${IMAGE_NAME}:latest ; fi
+
+      - run:
+          name: Run integration tests # TODO: double check this base_url value
+          command: |
+            docker run --entrypoint /bin/bash --workdir "/martha" ${IMAGE_NAME}:${CIRCLE_BRANCH} -c "BASE_URL=\"https://martha-fiab.dsde-dev.broadinstitute.org:32443/dockerized-martha/us-central1\" npm run-script integration"
```

Comment:
As the comment above indicates, this is the URL that I was using while testing against my FiaB, but I need to double check that this is going to be the correct URL for circleci to use after building the new image